### PR TITLE
Bug 1993002: Bump API to fix kubebuilder directives

### PR DESCRIPTION
**Pull request by candita**
Bump API to fix kubebuilder directives from PRs
- https://github.com/openshift/api/pull/988
- https://github.com/openshift/api/pull/990

### CODE REVIEW

**Comment by candita** [comment]
Out of memory issue:
`Aug 18 21:42:56.078 E ns/e2e-test-prometheus-6hrlk pod/execpod node/ip-10-0-178-217.us-west-1.compute.internal container/agnhost-container reason/ContainerExit code/137 cause/Error 
`
/test e2e-aws

File: go.mod [hunk 1]
```diff
@@ -1,28 +1,32 @@
...
 	gopkg.in/warnings.v0 v0.1.1 // indirect
-	k8s.io/api v0.21.1
-	k8s.io/apimachinery v0.21.1
-	k8s.io/client-go v0.21.1
-	k8s.io/component-base v0.21.1
-	k8s.io/klog/v2 v2.8.0
+	k8s.io/api v0.22.0
+	k8s.io/apimachinery v0.22.0
+	k8s.io/client-go v0.22.0-rc.0
+	k8s.io/component-base v0.22.0-rc.0
```
Comment (aojea) [inline go.mod:27]: should we be consistent and have the same version in all the dependencies?, in this case v0.22.0

Comment (sttts) [reply go.mod:27]: yes, we should

Comment (sttts) [reply go.mod:27]: yes, we should

Comment (candita) [reply go.mod]: Done, thanks.

**Comment by candita** [comment]
`sig-storage] CSI Volumes [Driver: csi-hostpath] [Testpattern: Pre-provisioned PV (ext3)] volumes should allow exec of files on the volume [Suite:openshift/conformance/parallel] [Suite:k8s]`
/test e2e-aws

**Pull request merged**